python/pyspark/py.typed

```diff
@@ -0,0 +1 @@
+partial
```

Comment:
quick question: what is this?

Reply:
In case of stub packages this allows us to skip annotations for certain modules ‒ if stub file is not found, type checker will go and inspect the module itself.

In theory, if stub files reside within the same directory structure as the source, it shouldn't be necessary.  But since things are not always smooth, I'd prefer to keep this.